_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -9180,8 +9180,8 @@ mfxU32 HeaderPacker::WriteSlice(
             obs.PutBit(IsOn(m_directSpatialMvPredFlag));
         if (sliceType != SLICE_TYPE_I)
         {
-            mfxU32 numRefIdxL0ActiveMinus1 = MFX_MAX(1, task.m_list0[fieldId].Size()) - 1;
-            mfxU32 numRefIdxL1ActiveMinus1 = MFX_MAX(1, task.m_list1[fieldId].Size()) - 1;
+            mfxU32 numRefIdxL0ActiveMinus1 = std::max<mfxU32>(1, task.m_list0[fieldId].Size()) - 1;
+            mfxU32 numRefIdxL1ActiveMinus1 = std::max<mfxU32>(1, task.m_list1[fieldId].Size()) - 1;
```

Comment:
If Size() returns 0 the result is 0xffffffff. Similar below.

Reply:
no, since this is ```max```, and result will be 0